tests/exploreServices.spec.ts

```diff
@@ -1,6 +1,7 @@
 import { test, expect } from '@grafana/plugin-e2e';
 import { ExplorePage } from './fixtures/explore';
 import {testIds} from "../src/services/testIds";
+import {mockVolumeApiResponse} from "./mocks/mockVolumeApiResponse";
```

Comment:
I know it's tests, but those imports look different in formatting.

Reply:
oh yeah, the linter doesn't run in the test files, I'll fix!